docs/source/examples/combine_techniques.md

```diff
@@ -0,0 +1,107 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Composing techniques: Readout Error Mitigation and Zero Noise Extrapolation
+
+Noise in quantum computers can arise from a variety of sources, and sometimes applying multiple error mitigation techniques can be more beneficial than applying a single technique alone. Here we apply a combination of readout error mitigation (REM) and zero noise extrapolation (ZNE) to a randomized benchmarking (RB) task. In REM, the inverse transition / confusion matrices are generated and applied to the noisy measurement results. In ZNE, the expectation value of the observable of interest is computed at different noise levels, and subsequently the ideal expectation value is inferred by extrapolating the measured results to the zero-noise limit. More information on the [REM](../guide/rem.md) and [ZNE](../guide/zne.md) techniques can be found in the corresponding sections of the user guide.
```

Comment:
- I think we typically stylize mitigation techniques with capital letters, so Zero-Noise Extrapolation.
- I really like how you added references for further info about REM/ZNE, but I think it makes sense to move those links to the place they first appear in case a reader hasn't heard of them before. It would probably eliminate the need for the last sentence as well.
- I think there should only be one inverse confusion matrix, not many.

---

Since this paragraph is set on a single line, it makes it hard to make review comments accurately. Hopefully I was able to communicate the sections I think could be modified, but I think using a line-character limit, or a single sentence per line rule[^1] could be helpful.

[^1]: my preferred way.

Reply:
All good points! I believe I've addressed them all.